### ISSUE TITLE
Incorrect indentation and space detection when using OR operator

### ISSUE THREAD

**Issue by nosuchip** [duaraghav8/Ethlint#223]
Example contract:

```
pragma solidity ^0.4.17;

contract A {
    function pack(uint32 param1, uint16 param2, uint32 param3) public pure returns(bytes32) {
        return bytes32(
            (uint256(param1) << 128) |
            (uint256(param2) << 64) |
            (uint256(param3) << 32)
        );
    }
}
```
`soliumrc.json`:

```json
{
  "extends": "solium:recommended",
  "plugins": ["security"],
  "rules": {
    "quotes": ["error", "double"],
    "indentation": ["error", 4]
  }
}
```

Execution:
```shell
alp@mobby ~/proj/smartcontracts $ ./node_modules/.bin/solium --version
Solium version 1.1.7
alp@mobby ~/proj/smartcontracts $ ./node_modules/.bin/solium --file indent.sol

indent.sol
  6:12    error      Only use indent of 12 spaces.                                        indentation
  6:35    warning    Single space should be either on both sides of '|' or not at all.    operator-whitespace

No fixes were applied.
✖ 1 error, 1 warning found.
```

Specifying `--fix` flag does not actually fix the file. Putting all ORed expressions into the single line fixes the error.

**Comment by duaraghav8** [comment]
Thanks @nosuchip for reporting this!
The operator-whitespace issue is a known bug but the indent one is new.
As mentioned in the [docs](http://solium.readthedocs.io/en/latest/user-guide.html#automatic-code-formatting), solium doesn't fix all the issues, although spacing & indent fixes are on the roadmap.

**Comment by nosuchip** [comment]
@duaraghav8 thanks for reply. I tried to use `fix` as last resort - to understand what formatting is suitable for solium.

**Comment by duaraghav8** [comment]
see http://solium.readthedocs.io/en/latest/user-guide.html#list-of-style-rules for which rules provide fix functionality

**Comment by nosuchip** [comment]
Already tried it. Only way to fix this warning is to join evrything under `bytes32()` to the single line. Meanwhile if change function return type to `uint256` and remove explicit conversion to `bytes32` warning about indentation fixed. IDK how it is related to type specification.

**Comment by duaraghav8** [comment]
Resolved in `v1.2.2`.
Sorry this took way too long, but its done now!
